docs/installation.md

```diff
@@ -60,3 +60,16 @@ You should only use these during the development phase of your project, never in
 ```html
 <script src="https://cdn.jsdelivr.net/npm/@nebula.js/stardust@0.6.0/dist/stardust.dev.js" crossorigin></script>
 ```
+
+## Linking Nebula packages to your own repository
+
+When you use Nebula.js as a dependency in your project, and you want to edit or debug the Nebula.js source code, you can use either `npm link` or `yarn link` to accomplish this.
+
+### Linking Nebula when using `nebula serve`
+
+With `npm`: Before using `nebula serve` to serve your project's files, you must run `npm link` in the folder: `nebula.js/commands/serve` and use it in your own repo by running `npm link @nebula.js/cli-serve` (run this in your own repo).
+To include source maps, so you can easier debug the Nebula.js code, you should also run `yarn build:dev` in the `nebula.js/commands/serve` folder.
+
+With `yarn`: Same as for `npm` but replace `npm` with `yarn` in the commands above.
+
+In your browser's debugging tool you should now be able to see the Nebula.js source code and to add breakpoints inside of the Nebula.js files.
```

Comment:
```suggestion
## Linking nebula packages to your own repository

When you use nebula.js as a dependency in your project, and you want to edit or debug the nebula.js source code, you can use either `npm link` or `yarn link` to accomplish this.

### Linking nebula when using `nebula serve`

With `npm`: Before using `nebula serve` to serve your project's files, you must run `npm link` in the folder: `nebula.js/commands/serve` and use it in your own repo by running `npm link @nebula.js/cli-serve` (run this in your own repo).
To include source maps, so you can easier debug the nebula.js code, you should also run `yarn build:dev` in the `nebula.js/commands/serve` folder.

With `yarn`: Same as for `npm` but replace `npm` with `yarn` in the commands above.

In your browser's debugging tool you should now be able to see the nebula.js source code and to add breakpoints inside of the nebula.js files.
```

Reply:
I'm no English language expert but isn't `nebula` a cosmic gas cloud and `Nebula` the name of the package 🤔 
But since `nebula` is used elsewhere we should be consistent.